mcs/class/corlib/corlib_xtest.dll.sources

```diff
@@ -145,3 +145,4 @@
 ../../../external/corefx/src/System.Text.Encoding/tests/EncodingTestHelpers.cs
 ../../../external/corefx/src/System.Text.Encoding/tests/NegativeEncodingTests.cs
 
+../../../external/corefx/src/System.Memory/tests/*.cs
```

Comment:
this won't include the subdirectories so it'll only include AllocationHelper.cs and TInt.cs